### ISSUE TITLE
Autocompletion breaking at accent

### ISSUE THREAD

**Issue by Bouzmine** [atom/autocomplete-plus#699]
When trying to autocomplete words that are not functions or variables, I found out that the autocompletion breaks just before an accent ((e.g. numéro -> num). As I am a French speaker, this is an issue as most of the words I want to autocomplete contain accents.

**Comment by Aklal** [comment]
The same in german

**Comment by fiapps** [comment]
Until recently, this could be fixed by checking the "Enable Extended Unicode Support" option in the package settings. However, now it has stopped working.

**Comment by leroix** [comment]
Thanks for the report. I was able to reproduce the behavior you're seeing in 1.22.1.

There was a fix for this, and I believe it should land in 1.23.

**Comment by fiapps** [comment]
I'm still seeing this in 1.23.1.

**Comment by fiapps** [comment]
Using atom 1.27.2 (autocomplete plus 2.40.6), I tracked down the issue in my case to the setting of the `LANG` environment variable.

For some time (maybe 2 years), I've been running `env LANG=it-IT atom foo.md` to get spell checking for that language when I write in it. When I do this, autocompletion breaks as described by the OP. I've discovered that if I don't set it (`atom foo.md`), autocompletion of accented words works. Further testing revealed that it doesn't work with `LANG=it-IT.UTF-8` either, but it does work with `LANG=it_IT.UTF-8` (note the underscore).

I've also discovered that the spell checker now has a setting for the locale(s) to use, so I don't need to set `LANG`.
